packages/phpstan-extensions/src/Testing/AbstractServiceAwareRuleTestCase.php

```diff
@@ -14,7 +14,7 @@
  * @template TRule of \PHPStan\Rules\Rule
  * @template-extends RuleTestCase<TRule>
  *
- * @deprecated Use getAdditionalConfigFiles() RuleTestCase and self::getContainer->getByType() instead
+ * @deprecated Use getAdditionalConfigFiles(), PHPStan\Testing\RuleTestCase and self::getContainer()->getByType() instead
```

Comment:
RuleTestCase is technically correct as in use statement, that will need `\` prefix for fqcn if going to be used